tests/test_expectation_maximization.py

```diff
@@ -0,0 +1,173 @@
+"""Test bayesian expectation maximization process."""
+
+import numpy as np
+import pytest
+
+from reconstructSPI.iterative_refinement import expectation_maximization as em
+
+
+@pytest.fixture
+def n():
+    """Get test array size value for consistency."""
+    return 2
+
+
+@pytest.fixture
+def test_ir(n):
```

Comment:
Yes, I think writing this upfront is a nice conscience way. I see how it's reused, including in `test_iterative_refinement`